dipy/data/files/meson.build

```diff
@@ -14,6 +14,7 @@ data_files = [
     'dki_constraint_2.npz',
     'dki_constraint_4.npz',
     'dki_constraint.npz',
+    'dki_constraint_SC.npz',
```

Comment:
Maybe instead of your initials, we could name this with a word that explains how these are different?

Reply:
Could do, I don't think it matters very much though. I though having a second file would be better, so that the old one wasn't deleted.

I have a paper under review that actually shows and explains these equations, so my initials are not the worst choice. The main difference with the existing ones are just extra slack variables and different form of the main basis matrix - it appears to do exactly the same as before, but this one is more general because is it fully parameterized with all the possible slack matrices.

What would you suggest? Can I just leave a comment instead?

Reply:
If this isn't that important, can we leave until I get a chance to write the tutorial?